lib/daru/index/index.rb

```diff
@@ -44,24 +44,15 @@ def each(&block)
     end
 
     attr_reader :relation_hash, :size
+    attr_accessor :name
 
-    def initialize index
-      index =
```

Comment:
These index guesses is in other method [`guess_index`](https://github.com/SciRuby/daru/pull/323/files#diff-a1adf8860e167030bf548e2529c7d508R219), to maintain MethodLength.( otherwise rubocop error `Metrics/MethodLength`).